tests/generic_config_updater/test_pfcwd_interval.py

```diff
@@ -81,7 +81,9 @@ def get_detection_restoration_times(duthost):
     Args:
         duthost: DUT host object
     """
-     
+    
+    duthost.shell('config pfcwd start --action drop all 400 --restoration-time 400', module_ignore_errors=True)
+    time.sleep(15)
```

Comment:
>15 [](http://example.com/codeflow?start=15&length=2)

Why 15 seconds? I see other code sleep 5 seconds. Still is it possible not to use magic number at all? For example, sleep until we find a signal that the config take effect.
@neethajohn

Reply:
We do not need 15s. we can check immediately